Users/Install.rst

```diff
@@ -83,10 +83,77 @@ To install coala only (without any bears), you can do:
     **Ubuntu Users** - Do not use ``apt-get install coala``
     for installing coala as that is different software.
 
+Installing inside a pipenv
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pipenv is  user friendly method of managing virtual environments and
+packages.
+You can look at it as a mix of ``pip`` and ``virtualenv``. So you don't have
+to manage them seperately for your projects.
+
+First, assuming you have installed python and pip earlier from
+this guide we'll now install ``pipenv`` by running the following command.
+
+::
+
+    $ pip install pipenv
+
+Now we create a python 3 environment for our project, move to your project
```

Comment:
"**our** project, **your** project directory"
Should be unified a bit ;) either we are doing this "together with the user", or it's always the user's stuff :)

Reply:
User context would be more effective in this case imo.